src/App.js

```diff
@@ -1,24 +1,23 @@
-import logo from './logo.svg';
```

Comment:
- [ ] The structure of the files should follow the `ducks pattern` :arrow_down: , Kindly make this

```
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
```